src/components/shared/page.tsx

```diff
@@ -156,6 +157,17 @@ function ArticleBody({ capi, imageSalt, getAssetLocation }: BodyProps): ElementW
     const liveblogScript = getAssetLocation('liveblog.js');
     const mediaScript = getAssetLocation('media.js');
 
+    if (item.design === Design.Interactive && item.display === Display.Immersive) {
```

Comment:
Yes there is, they still expect page elements like the header to be there but because it's `interactive` the interactive atoms won't be iframed so their styles can override page styles.

This is starting to explain some of the issues we have. Our page structure is very different to dotcom (classnames and elements on the page). So interactive developers would have to add lots of unique styles for apps by targeting ".ios" or "android"


Immersive interactives should be more of a blank page, where the atoms provide their own header. standfirst etc.

Reply:
Actually according to @duarte `interactive` articles are more common than `immersive interactives`

Reply:
@JamieB-gu one thing to look out for is that many Immersive articles are not Interactive — so they're independent switches: Interactive is a composer file type, while Immersive is a display hint available to both standard articles and Interactives

Reply:
So in the context of these changes, a piece of content that is `Design.Interactive` but _isn't_ `Display.Immersive` won't render as an interactive? @webb04